yt_dlp/extractor/chaturbate.py

```diff
@@ -41,10 +42,54 @@ class ChaturbateIE(InfoExtractor):
     }]
 
     _ROOM_OFFLINE = 'Room is currently offline'
+    _ROOM_PRIVATE = 'Room is currently in a private show'
+    _ROOM_AWAY = 'Performer is currently away'
+    _ROOM_PASSWORD = 'Room is password protected'
+    _ROOM_HIDDEN = 'Hidden session in progress'
+    _ROOM_BLOCKED = 'Room is not avaiable in this region'
```

Comment:
let's condense this into a single class var mapping (and remove `_ROOM_BLOCKED` since we won't need it per my suggestion below)
```suggestion
    _ERROR_MAP = {
        'offline': 'Room is currently offline',
        'private': 'Room is currently in a private show',
        'away': 'Performer is currently away',
        'password protected': 'Room is password protected',
        'hidden': 'Hidden session in progress',
    }
```

&nbsp;
then in `_extract_from_webpage` we'll need to adjust this:
```diff
             if not error:
                 if any(p in webpage for p in (
-                        self._ROOM_OFFLINE, 'offline_tipping', 'tip_offline')):
+                        self._ERROR_MAP['offline'], 'offline_tipping', 'tip_offline')):
-                    error = self._ROOM_OFFLINE
+                    error = self._ERROR_MAP['offline']
```